UndertaleModTool/Scripts/Community Scripts/ExternalizeAllOGGs.csx

```diff
@@ -28,35 +28,32 @@ Otherwise, select 'No', and make a backup of the game before using this script.
 }
 
 //Overwrite Folder Check One
-if (Directory.Exists(winFolder + "Exported_Sounds\\"))
+string exportedSoundsPath = Path.Combine(winFolder, "Exported_Sounds");
+if (Directory.Exists(exportedSoundsPath))
 {
     bool overwriteCheckOne = ScriptQuestion(@"An 'Exported_Sounds' folder already exists.
 Would you like to remove it? This may some time.
 
 Note: If an error window stating that 'the directory is not empty' appears, please try again or delete the folder manually.
 ");
-    if (overwriteCheckOne)
-        Directory.Delete(winFolder + "Exported_Sounds\\", true);
     if (!overwriteCheckOne)
     {
         ScriptError("An 'Exported_Sounds' folder already exists. Please remove it.", "Error: Export already exists.");
         return;
     }
+    
+    Directory.Delete(exportedSoundsPath, true);
 }
 
 // Group by audio group check
-var groupedExport = 0;
+bool groupedExport;
```

Comment:
```suggestion
bool groupedExport = false;
```

Reply:
False is the default value of booleans. Any specific reason you want it explicitly mentioned?

Reply:
Yeah, nevermind, I confused it with `string`.